aequilibrae/paths/linear_approximation.py

```diff
@@ -0,0 +1,368 @@
+import numpy as np
```

Comment:
I don't like reviewing code I've written, even though you've parallelised it quite a bit (nice work btw, really impressed with performance compared to the initial prototype). I'll try to get some multi-class assignment tests going, though.

Reply:
Cheers!

Reply:
I have reviewed the code while I was refactoring, so it might be good enough between the two of us.